docs/customizing.md

```diff
@@ -245,3 +245,271 @@ Example `pulp_install_plugins` (with bogus version values):
       pulp-rpm:
          version: "7.8.9"
 ```
+
+Deployment Scenarios
+--------------------
+
+In this section, you can find information and Ansible playbook examples that demonstrate how to configure the following installation scenarios:
+
+* A single “all-in-one” Pulp server that includes the database, redis, and webserver.
+* Pulp on one server, while using existing servers for the database, redis, and webserver.
+* Pulp on one server, with the database, redis & webserver on separate servers.
+* Each and every Pulp service on a separate server.
+
+Depending on your requirements for high availability and better scalability, you might like to customize your installations according to these examples.
+
+
+#### 'All in One Pulp Server' Example
+
+This deployment consists of installing a single server that contains all Pulp services, including the database, redis & webserver.
+
+```
+---
+- hosts: example-pulp-server
+  vars:
+    pulp_default_admin_password: << YOUR PASSWORD FOR THE PULP APPLICATION HERE >>
+    pulp_settings:
+      secret_key: << YOUR SECRET HERE >>
+      content_origin: "http://"
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+  roles:
+    - pulp_all_services
+  environment:
+    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+```    
+
+#### Pulp Server with Existing Infrastructure
+
+This deployment consists of a single Pulp server that relies on an existing database, redis and webservers.
+
+```
+---
+- hosts: example-pulp-server
+  vars:
+    pulp_default_admin_password: << YOUR PASSWORD FOR THE PULP APPLICATION HERE >>
+    pulp_content_bind: example-pulp-server:24816
+    pulp_api_bind: example-pulp-server:24817
+    pulp_settings:
+      secret_key: << YOUR SECRET HERE >>
+      content_origin: "http://exanple-webserver.fqdn"
+      redis_host: example-redis-server
+      redis_port: 6379
+      redis_password: << YOUR REDIS PASSWORD HERE >>
+      databases:
+        default:
+          HOST: example-postgres-server
+          ENGINE: django.db.backends.postgresql_psycopg2
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+  roles:
+    - pulp_services
+  environment:
+    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+```
+
+#### Pulp with Separate Servers for Services
+
+This scenario has the following layout:
+
+* Pulp on one server
+* The database on a second server
+* redis on a third server
+* The webserver on a fourth server
+
+Note that in the following example `pulp_webserver` still depends on `pulp_common`.
+`pulp_common` provides the per-plugin webserver snippets.
+This means that the webserver will have pulpcore and all plugins installed on it.
```

Comment:
It will have the python packages installed but not be running the code, right?

Reply:
I pulled from this note: 

Note that the following non-SOA behavior still exists:

    pulp_webserver still depends on pulp_common (formerly pulp), so as to provide the per-plugin webserver snippets. So webserver1 will have pulpcore & all plugins installed on it. Now however, pulp-api is no longer running & listening by default (no need to set pulp_install_api_service=false.)

Reply:
I just paraphrased from the original blog. It could be outdated: 

Note that the following non-SOA behavior still exists:

    pulp_webserver still depends on pulp_common (formerly pulp), so as to provide the per-plugin webserver snippets. So webserver1 will have pulpcore & all plugins installed on it. Now however, pulp-api is no longer running & listening by default (no need to set pulp_install_api_service=false.)

Reply:
I changed this one.. hopefully for the better.

Reply:
Yes, re-architecting the pulp_webserver role is something on my long-term to-do-list for pulp_installer.

In the meantime, this doc change looks good.